release.js

```diff
@@ -36,15 +36,17 @@ module.exports = async function ({ github, context, inputs }) {
 
   const { opticUrl, npmTag, version, id } = releaseMeta
 
+  const run = runSpawn()
   if (!pr.merged) {
-    return github.rest.repos.deleteRelease({
+    await run('git', ['push', 'origin', '--delete', `"release/${version}"`])
+
+    return await github.rest.repos.deleteRelease({
```

Comment:
let's do these operations in a way that the failure of one doesn't cause the failure of the other. it would also be good if we could log, when something goes wrong, what went wrong